_includes/v23.1/known-limitations/cdc-queries.md

```diff
@@ -0,0 +1,9 @@
+- You can only apply CDC queries on a single table in each statement.
+- Some [stable functions](functions-and-operators.html#built-in-functions), notably functions that return MVCC timestamps, are overridden to return the MVCC timestamp of the event.
```

Comment:
Let's give explicit examples `transaction_timestamp` and `statement_timestamp`.

In addition, some time based functions -- such as `now()` are disallowed because they are confusing in the CDC context -- use transaction_timestamp or crdb_internal_mvcc_timestamp column instead.

Reply:
Added this detail